### PR TITLE
Use r-* packages instead of vendoring within build

### PR DESCRIPTION
**Pull request by thompsonsed**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,29 +10,47 @@ source:
   sha256: 1ee490a1b7a0e1164fda08d38053a17f974f82f0237b8c2864bd12a28f3a8efb
 
 build:
-  number: 8
+
+
+  number: 16
+
   skip: True  # [(win and vc < 14)]
   skip: True  # [(win and py>=38)]
```
Comment (chrisburr) [inline recipe/meta.yaml:18]: I think you're missing this:
```suggestion
  skip: True  # [(win and py>=38)]
  script: {{ PYTHON }} -m pip install . -vv 
```

File: recipe/build.sh [hunk 2]
```diff
@@ -1,4 +1 @@
-if [[ $(uname) != Darwin ]] ; then
-    TZ="Europe/London" Rscript ${RECIPE_DIR}/build.R
-fi
-${PYTHON} -m pip install . -vv
\ No newline at end of file
+"%PYTHON%" -m pip install . -vv
```
Comment (chrisburr) [inline recipe/build.sh]: This is the Windows batch variable syntax:
```suggestion
"${PYTHON}" -m pip install . -vv
```
Alternatively you can delete the `recipe/bld.bat` and `recipe/build.sh` scripts and use my other suggestion.

Comment (thompsonsed) [reply recipe/build.sh]: Thanks, I'll delete the separate scripts and use your earlier suggestion.

**Comment by chrisburr** [comment]
@conda-forge-admin, please rerender.

**Comment by isuruf** [comment]
Thanks @thompsonsed for fixing this. I noticed that `r-wgcna` was not added as a dependency. Do you need help in adding that to conda-forge?

File: recipe/meta.yaml [hunk 3]
```diff
@@ -10,39 +10,51 @@ source:
...
+    - r-base
+    - r-nlme
+    - r-hmisc
+    - r-rcpp >=0.11.0
+    - r-doparallel
+    - r-dynamictreecut >=1.62
+    - r-fastcluster
+    - r-foreach
+    - r-matrixstats >=0.8.1
+    - r-survival
```
Comment (isuruf) [inline recipe/meta.yaml]: You don't need any of this in `host`. Just run
```suggestion
```

File: recipe/meta.yaml [hunk 4]
```diff
@@ -10,39 +10,51 @@ source:
...
   
 requirements:
   build:
-    - {{ compiler('c') }}  # [not osx]
-    - {{ compiler('cxx') }}  # [not osx]
-    - {{ compiler('fortran') }}  # [not osx]
+    - {{ compiler('c') }}  # [not win]
+    - {{ compiler('m2w64_c') }}  # [win]
+    - {{ compiler('cxx') }}  # [not win]
+    - {{ compiler('m2w64_cxx') }}  # [win]
```
Comment (isuruf) [inline recipe/meta.yaml]:
```suggestion
```

**Comment by thompsonsed** [comment]
> Thanks @thompsonsed for fixing this. I noticed that `r-wgcna` was not added as a dependency. Do you need help in adding that to conda-forge?

Yes that's a good idea. I'll create a pull request for a new feedstock. If I follow the [conda-forge helper script for R](https://github.com/bgruening/conda_r_skeleton_helper) is that the recommended approach?

**Comment by isuruf** [comment]
> If I follow the conda-forge helper script for R is that the recommended approach?

Yes, it is.

**Comment by thompsonsed** [comment]
Actually, I've just noticed that some of the R package dependencies for r-wgcna have been moved to Bioconductor (e.g. impute, preprocessCore, GO.db).

Does this mean that r-wgcna (and by extension iterativeWGCNA) should be on bioconda instead of conda-forge? I found the following in the bioconda docs:

> Most R packages on CRAN should be submitted at Conda-Forge. Specifically, if the CRAN package has a Bioconductor package dependency, it belongs in Bioconda. If the CRAN package does not have a Bioconductor package dependency, it belongs in Conda-Forge.

Is the correct thing to do therefore to migrate the package to Bioconda? Thanks @isuruf for your help with this.

**Comment by isuruf** [comment]
cc @conda-forge/bioconda-recipes

**Pull request merged**